.evergreen/config.yml

```diff
@@ -205,20 +213,17 @@ functions:
       type: test
       params:
         working_dir: astrolabe-src
+        shell: bash
         # Next commands will exit with appropriate status
         continue_on_err: true
         env:
           ATLAS_PROJECT_NAME: ${ATLAS_PROJECT_NAME}
           ATLAS_PROJECT_BASE_NAME: ${project}
-          CLUSTER_NAME_SALT: ${build_id}
-          ATLAS_API_USERNAME: ${atlas_key}
-          ATLAS_API_PASSWORD: ${atlas_secret}
-          ATLAS_API_BASE_URL: ${atlas_url}
-          ATLAS_ORGANIZATION_ID: ${atlas_organization_id}
           ATLAS_ADMIN_API_USERNAME: ${atlas_admin_api_username}
           ATLAS_ADMIN_API_PASSWORD: ${atlas_admin_api_password}
-        add_expansions_to_env: true
```

Comment:
This command needs `DRIVER_DIRNAME` to be set. Otherwise, the workload executors fail with errors like:
```
[2024/10/30 18:53:06.610] /astrolabe-src/integrations/go/workload-executor: line 5: ./integrations//executor: No such file or directory
```